### PR TITLE
[Docs] Add an example configuration to use dagger with TravisCi

### PR DESCRIPTION
**Pull request by jjuarez**
This is a really easy proposal of how can we use dagger in TravisCi, the example tried to follow the features exposed in the other examples, like the use of an specific version, cache, etc

### CODE REVIEW

File: docs/getting-started/1201-ci-environment.md
Comment (marcosnils) [inline docs/getting-started/1201-ci-environment.md:49]: this variable is not being used anywhere?

**Review by marcosnils — COMMENTED** [review]
Added some merge non-blocking changes.

I've realized that our `Gitlab` CI guide also has the same items I've highlighted here. I think I'll defer this to @jpadams to take a final decision.

File: docs/getting-started/1201-ci-environment.md
Comment (jjuarez) [reply docs/getting-started/1201-ci-environment.md:49]: Yes, because is defined at the worker level, it acts like a regular environment variable

File: docs/getting-started/1201-ci-environment.md
Comment (jjuarez) [reply docs/getting-started/1201-ci-environment.md:80]: It could be a better idea to reduce the complexity of this command, and also the pipeline, at the end this should be an example, right?

**Comment by jpadams** [comment]
@jjuarez The only change we need before merge is to make sure you use the preferred capitalization and spacing of "Travis CI" per their website.
https://github.com/dagger/dagger/pull/2702/files#diff-e309255ddd52101788f3de79596d0b93496c72d4b04059e44154f9e12b63e4d0R23

Also, could you make sure these extra files are not in the commit?
image

**Comment by jjuarez** [comment]
> @jjuarez The only change we need before merge is to make sure you use the preferred capitalization and spacing of "Travis CI" per their website. https://github.com/dagger/dagger/pull/2702/files#diff-e309255ddd52101788f3de79596d0b93496c72d4b04059e44154f9e12b63e4d0R23
> 
> Also, could you make sure these extra files are not in the commit? image

Sorry, are you talking about all the sliving directories of `pkg/universe.dagger.io/alpha`

**Review by jpadams — APPROVED** [review]
LGTM! Thanks for the changes @jjuarez !

**Pull request merged**
